server/resources/resource_management_test.go

```diff
@@ -417,8 +459,8 @@ var _ = Describe("Resource manager", func() {
 					result := context["response"].(map[string]interface{})
 					number := context["total"].(uint64)
 					Expect(err).NotTo(HaveOccurred())
-					Expect(number).To(Equal(uint64(2)))
-					Expect(result).To(HaveKeyWithValue("tests", ConsistOf(adminResourceData, memberResourceData)))
+					Expect(number).To(Equal(uint64(3)))
+					Expect(result).To(HaveKeyWithValue("tests", ConsistOf(adminResourceData, memberResourceData, otherDomainResourceData)))
```

Comment:
If `adminAuth` means super-user, this result should be expected. However, if `adminAuth` is not super-user, the result should be only `adminResourceData` here, because token is tenant-scoped.